hack/udevd/90-selinux.rules

```diff
@@ -0,0 +1,11 @@
+SUBSYSTEM=="*",SECLABEL{selinux}="system_u:object_r:device_t:s0"
```

Comment:
are these labels like standard? I guess devices can have standard labels

Reply:
Apparently `(fsuse trans devtmpfs (system_u object_r device_t (systemLow systemLow)))` is not sufficient for this to be default